app/views/RoomView/index.js

```diff
@@ -390,6 +390,11 @@ class RoomView extends React.Component {
 		const observable = room.observe();
 		this.subSubscription = observable
 			.subscribe((changes) => {
+				const { navigation } = this.props;
+				const name = navigation.getParam('name');
+				if (changes.name !== name) {
+					navigation.setParams({ name: changes.name });
```

Comment:
I think we're missing other title headers here.
If it's a discussion or Real Name is enabled, the header comes from `fname`.
Check it out: https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/views/RoomsListView/index.js#L484

Reply:
Thanks for the review Sir. So first I should check whether name param is having some value or not, if there is no value, then I should use fname param, am I right on this?